src/components/TreeReactFlow/index.tsx

```diff
@@ -355,7 +459,8 @@ const makePrimerNode = async (
   p: NodeParams,
   layout: LayoutParams,
   zIndex: number,
-  nodeType: NodeType
+  nodeData: NodeData,
+  def?: GlobalName
```

Comment:
We would ideally modify types somehow to represent that we always expect this to be present for "normal" trees (i.e. not eval), and leaving it off means no selection will be indicated.